python/lsst/cp/pipe/cpFunctions.py

```diff
@@ -0,0 +1,516 @@
+# This file is part of cp_pipe.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If
+
+# Utility tasks/methods copied from pipe_drivers/constructCalibs.py
+
+import datetime
+import numpy as np
+import time
+
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+import lsst.pipe.base.connectionTypes as cT
+import lsst.afw.math as afwMath
+import lsst.afw.image as afwImage
+import lsst.daf.base as dafBase
+
+
+class BlessCalibration(pipeBase.Task):
+    """Create a way to bless existing calibration products.
+
+    The inputs are assumed to have been constructed via cp_pipe, and
+    already exist in the butler.
+
+    Parameters
+    ----------
+    kwargs : `dict`
```

Comment:
Why specify this as `**kwargs` instead of actual values? I think this would be better:

    def __init__(self, *, butler, inputCollection, outputCollection, **kwargs):

But then `kwargs` would be unused. Why is there no `Task.__init__(**kwargs)`?